pkg/docker/container.go

```diff
@@ -72,7 +72,7 @@ func EnsureContainer(ctx context.Context, log *zap.SugaredLogger, cli *client.Cl
 
 		err := cli.ContainerStart(ctx, ci.ID, types.ContainerStartOptions{})
 		if err != nil {
-			log.Errorw("starting container failed", "container_id", container.ID)
+			log.Errorf("starting container failed", "container_id %s: %v", ci.ID, err)
```

Comment:
Not related to this change, but when I encountered this, `container.ID` caused a nil pointer panic.

Reply:
`func (s *SugaredLogger) Errorf(template string, args ...interface{}) {`

`Errorf` is taking template as a first argument and then args after that. If `container.ID` is nil, maybe just use `opts.ContainerName` in the `log.Errorw` or fix the `log.Errorf` arguments.

Reply:
Changed from ci.ID to opts.ContainerName.